repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/system-model/ignore.bindings

```diff
@@ -18,17 +18,12 @@
   limitations under the License.
   =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
   -->
-
-
-<bindings xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://java.sun.com/xml/ns/jaxb"
+<bindings xmlns="http://java.sun.com/xml/ns/jaxb"
```

Comment:
we just want to delete this file don't we?

Reply:
Nuke it

Reply:
Gone.